folding/utils/logging.py

```diff
@@ -80,5 +83,13 @@ def log_event(self, event):
     if not getattr(self, "wandb", None):
         init_wandb(self)
 
+    if getattr(self, "wandb_run_start", None):
```

Comment:
this will never be false if the init_wandb method was run above (which it will). I don't think you need this if

Reply:
True, this was a left over from a different version of the solution where I implemented this only for validators and not miners